pipeline/cf_value.go

```diff
@@ -3,12 +3,15 @@ package pipeline
 import "reflect"
 
 type (
+	/*
+		GoStructorValue - it's main type which using by this library. In this type setuping current preparing field and if field doesn't preparing in this type will be inserted special noValue interface.
```

Comment:
It is not understandable, rewrite it please

Reply:
Or explain it to me and we will write it together

Reply:
i changed that